tests/unit/src/test/scala/tests/codeactions/BaseCodeActionLspSuite.scala

```diff
@@ -15,21 +15,28 @@ abstract class BaseCodeActionLspSuite(suiteName: String)
       expectedActions: String,
       expectedCode: String,
       selectedActionIndex: Int = 0,
-      expectNoDiagnostics: Boolean = true
+      expectNoDiagnostics: Boolean = true,
+      fileName: Option[String] = None,
```

Comment:
```suggestion
      fileName: String = "A.scala",
```
should simplify it a bit.